### PR TITLE
Merge netmaster and auth_proxy

### PR DESCRIPTION
**Pull request by neelimamukiri**
This also cleans up most of the unused mounts for netmaster & netplugin.
Removing the `-x` flag for netplugin. Lets deal with cleanup/upgrades separately.

This is the first part of HA implementation for Contiv on K8s. TBD is etcd HA - I am still trying to understand/decide whether to go with StatefulSets or use the etcd operator for this.

This resolves https://github.com/contiv/ccn/issues/95 and https://github.com/contiv/ccn/issues/319.

### CODE REVIEW

File: install/k8s/k8s1.6/contiv.yaml [hunk 1]
```diff
@@ -273,88 +260,17 @@ spec:
                 configMapKeyRef:
                   name: contiv-config
                   key: config
-          securityContext:
```
Comment (dseevr) [inline install/k8s/k8s1.6/contiv.yaml]: Why are we able to drop the privileged requirement here?

Comment (neelimamukiri) [inline install/k8s/k8s1.6/contiv.yaml]: netmaster does not need to be a privileged container. It has k8s 1.6 RBAC to ask for the privileges it needs.

File: install/k8s/k8s1.6/contiv.yaml [hunk 2]
```diff
@@ -273,88 +260,17 @@ spec:
...
-      serviceAccountName: contiv-netmaster
-      containers:
         - name: contiv-api-proxy
           image: contiv/auth_proxy:__CONTIV_VERSION__
           args:
             - --tls-key-file=/var/contiv/auth_proxy_key.pem
             - --tls-certificate=/var/contiv/auth_proxy_cert.pem
             - --data-store-address=$(CONTIV_ETCD)
-            - --netmaster-address=__NETMASTER_IP__:9999
+            - --netmaster-address=localhost:9999
```
Comment (rchirakk) [inline install/k8s/k8s1.6/contiv.yaml]: >>localhost:9999
is it going to work when listen is URL specified for netmaster?

Comment (neelimamukiri) [inline install/k8s/k8s1.6/contiv.yaml]: That is part of https://github.com/contiv/install/pull/196 which is in review as well.

Comment (rchirakk) [inline install/k8s/k8s1.6/contiv.yaml]: 👍

**Pull request merged**